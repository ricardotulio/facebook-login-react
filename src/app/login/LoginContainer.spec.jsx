import React from 'react'
import renderer from 'react-test-renderer'

import LoginContainer from './LoginContainer'

test('Componet must be render as expected', () => {
  const loginContainer = renderer.create(<LoginContainer />)
  const tree = loginContainer.toJSON()

  expect(tree).toMatchSnapshot()
})