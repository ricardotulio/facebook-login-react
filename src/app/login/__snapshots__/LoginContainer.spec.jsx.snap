// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componet must be render as expected 1`] = `
<form>
  <input
    id="login"
    name="login"
    type="text"
  />
  <input
    id="password"
    name="password"
    type="password"
  />
  <button
    type="button"
  >
    SIGN IN
  </button>
</form>
`;
